sycl/test/basic_tests/esimd/global_var.cpp

```diff
@@ -0,0 +1,13 @@
+// RUN: %clangxx -fsycl -fsycl-explicit-simd -fsycl-device-only -fsyntax-only -Xclang -verify %s
+// expected-no-diagnostics
+
+#include <CL/sycl/intel/esimd.hpp>
+
+// This test checks that DPC++ compiler in ESIMD mode understands
+// the ESIMD_PRIVATE and ESIMD_REGISTER macros
```

Comment:
Not sure, this test makes sense in the SYCL dir, it basically checks frontend.

Reply:
This test includes esimd.hpp, so it can't reside in the FE